Source/qol.cpp

```diff
@@ -12,10 +12,25 @@
 namespace devilution {
 namespace {
 
+namespace xpbar {
+constexpr int barWidth = 307;
+constexpr int barHeight = 5;
+
+constexpr int goldGradient[] = { 0xCF, 0xCE, 0xCD, 0xCC, 0xCB, 0xCA, 0xC9, 0xC8, 0xC7, 0xC6, 0xC5, 0xC4 };
+constexpr int goldGrades = SDL_arraysize(goldGradient);

```

Comment:
Optional suggestion (in case you think it's better):

```suggestion
constexpr std::array<std::uint8_t, 12> = { 0xCF, 0xCE, 0xCD, 0xCC, 0xCB, 0xCA, 0xC9, 0xC8, 0xC7, 0xC6, 0xC5, 0xC4 };
```

Reply:
Sure, if it's OK that I drag the STL into this :)